Projects/19-Captsone_01/ispy1/inferential_statistics.py

```diff
@@ -0,0 +1,132 @@
+import pandas as _pd
+import numpy as np
+from scipy.stats import chi2_contingency as _chi2
+from scipy.stats import fisher_exact
+import matplotlib as plt
+import seaborn as sns
+from scipy import stats
+from scipy.stats import fisher_exact
+from sklearn import preprocessing
+from statsmodels.formula.api import ols
+import statsmodels.api as sm
+import matplotlib.pyplot as plt
+
+# CATEGORICAL PREDICTORS and CATEGORICAL OUTCOMES
+def contingency_table(predictor, outcome, dataframe):
+    '''
```

Comment:
OK. Perhaps we can discuss this on Thursday. I don't really get the benefits of different docstring method / conventions